eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -27,17 +27,33 @@
   <!-- Public API already released with incorrect upper-case naming pattern -->
   <suppress checks="MethodName" files="AsymmetricEncryptionAlgorithm.java"/>
   <suppress checks="MethodName" files="KeyWrapAlgorithm.java"/>
+  <suppress checks="MethodName" files="PartitionSenderImpl.java"/>
   <suppress checks="MethodName" files="SignatureEncoding.java"/>
   <suppress checks="MethodName" files="SymmetricEncryptionAlgorithm.java"/>
   <suppress checks="MethodName" files="RsaSignature.java"/>
 
   <!-- Public API already released with incorrect constant variable naming -->
   <suppress checks="ConstantName" files="AlgorithmResolver.java"/>
+  <suppress checks="ConstantName" files="AmqpErrorCode.java"/>
+  <suppress checks="ConstantName" files="BatchErrorCodeStrings.java"/>
+  <suppress checks="ConstantName" files="ConnectionStringBuilder.java"/>
```

Comment:
Looking over your PR #3105 , the checkstyle-suppression comments I had may be similar. I made the same comment about ConnectionStringBuilder.java being able to be removed.

Since you're merging 3105 after this, I'd suggest removing this line instead of fixing it.

Reply:
I updated checkstyle supression file. but not changes to the corresponding errors.